sklearn/feature_selection/tests/test_rfe.py

```diff
@@ -176,7 +178,8 @@ def test_rfecv():
 
     # same in sparse
     rfecv_sparse = RFECV(estimator=SVC(kernel="linear"), step=1, cv=5)
-    X_sparse = sparse.csr_matrix(X)
+    X_sparse = sparse.csc_matrix(X)
```

Comment:
According to the previous commits, it will  be better to use csc matrix, but within SVC estimator, ```check_array``` will convert it into csr matrix anyway.